docs/sphinx/using/backends/hardware.rst

```diff
@@ -596,4 +596,77 @@ the ``shots_count`` is set to 1000.
 
 To see a complete example for using Quantinuum's backends, take a look at our :doc:`Python examples <../examples/examples>`.
 
+QuEra Computing
+==================================
+
+.. _quera-backend:
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA-Q may access Aquila, QuEra's first generation of quantum
+processing unit (QPU) via AWS Braket. Hence, users must set AWS credentials using 
```

Comment:
Need to add a line about enabling AWS Braket.
```suggestion
processing unit (QPU) via AWS Braket. Hence, users must first enable Braket by 
following `these instructions <https://docs.aws.amazon.com/braket/latest/developerguide/braket-enable-overview.html>`__. 
Then set AWS credentials using any of the documented `methods <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html>`__.
```

Reply:
Also, for branding purposes, please use “Amazon Braket” rather than “AWS Braket”.